__e2e__/peruse.spec.js

```diff
@@ -115,6 +115,45 @@ describe( 'main window', () =>
         expect( text ).toBe( 'Invalid URL: http://:invalid-url');
     } );
 
+    if ( nodeEnv === 'dev' )
```

Comment:
@hunterlester do we need to block `prod` from running these tests?

Reply:
@joshuef This test fails if `NODE_ENV=prod` because it depends on the browser connecting to be able to make a web fetch. Intead of `example.com` returning `'Requested public name is not found'`, it will return `Setup Incomplete. Connection not available yet.`